kofta/public/locales/pt-BR/translation.json

```diff
@@ -1,211 +1,214 @@
+        "roomInviteFrom": "Convite para sala de",
+        "justStarted": "Acabaram de começar",
+        "likeToJoin": ", deseja entrar?",
+        "inviteReceived": "Você foi convidado para"
+      },
+      "editProfileModal": {
+        "usernameTaken": "Esse nome de usuário já existe.",
+        "avatarUrlError": "Imagem inválida",
+        "avatarUrlLabel": "Url da foto do Github/Twitter",
+        "displayNameError": "De 2 até 50 caracteres",
+        "displayNameLabel": "Nome de exibição",
+        "usernameError": "De 4 a 15 caracteres alfanuméricos/sublinhados",
+        "usernameLabel": "Usuário",
+        "bioError": "Máximo de 160 caractéres",
+        "bioLabel": "Biografia"
+      },
+      "profileModal": {
+        "blockUserConfirm": "Você tem certeza que quer bloquear esse usuário de entrar em qualquer sala que você criar?",
+        "blockUser": "Bloquear usuário",
+        "makeMod": "Tornar administrador",
+        "unmod": "Remover administrador",
+        "addAsSpeaker": "Adicionar como apresentador",
+        "moveToListener": "Mover para ouvinte",
+        "banFromChat": "Expulsar do chat",
+        "banFromRoom": "Expulsar da sala",
+        "goBackToListener": "Voltar para ouvinte",
+        "deleteMessage": "Excluir essa mensagem"
+      },
+      "roomSettingsModal": {
+        "requirePermission": "precisar de permissão para falar",
+        "makePublic": "fazer a sala pública",
+        "makePrivate": "fazer a sala privada"
+      }
+    },
+    "userVolumeSlider": { "noAudioMessage": "Não tem audio por algum motivo" },
+    "addToCalendar": { "add": "Adicionar ao Calendário" },
+    "keyboardShortcuts": {
+      "setKeybind": "Configurar atalho",
+      "listening": "Ouvindo",
+      "toggleMuteKeybind": "Atalho para mutar",
+      "togglePushToTalkKeybind": "Atalho para falar (push-to-talk)"
+    },
+    "wsKilled": {
+      "description": "O Websocket foi finalizado pelo servidor. Isso normalmente acontece quando você abre o site em outra aba.",
+      "reconnect": "Reconectar"
+    },
+    "deviceNotSupported": {
+      "notSupported": "O seu dispositivo não é suportado no momento. Você pode criar uma",
+      "linkText": "issue no GitHub",
+      "addSupport": "e eu tentarei adicionar suporte ao seu dispositivo."
+    },
+    "inviteButton": {
+      "invited": "Convidado(a)",
+      "inviteToRoom": "Convidar para a sala"
+    }
+  },
+  "modules": {
+    "scheduledRooms": {
+      "title": "Salas agendadas",
+      "noneFound": "Nenhuma sala encontrada",
+      "allRooms": "Todas as salas agendadas",
+      "myRooms": "Minhas salas agendadas",
+      "scheduleRoomHeader": "Agendar uma sala",
+      "startRoom": "Iniciar sala",
+      "modal": {
+        "needsFuture": "Precisa ser uma data no futuro",
+        "roomName": "Nome da sala",
+        "minLength": "O nome precisa ter pelo menos 2 caracteres"
+      }
+    },
+    "roomChat": {
+      "title": "Chat",
+      "emotesSoon": "[emotes soon]",
```

Comment:
Maybe translate this one?

Reply:
I don't think "chat" needs to be translated and didn't really know what he meant with "emotes soon" so I left it out. 
What do you think it should be?

Reply:
Agreed on the `Chat`, I've meant to point out only the `emotes soon`.
We could leave it as is or use something like `emotes logo`.